WebApp/autoreduce_webapp/selenium_tests/tests/test_run_summary.py

```diff
@@ -53,7 +54,8 @@ def test_opening_run_summary_without_run_variables(self):
         assert self.page.run_description_text() == "Run description: This is the test run_name"
 
 
-class TestRunSummaryPage(NavbarTestMixin, BaseTestCase, FooterTestMixin):
+class TestRunSummaryPage(BaseTestCase):
+    # class TestRunSummaryPage(NavbarTestMixin, BaseTestCase, FooterTestMixin):
```

Comment:
What is happening with this comment?

Reply:
I comment out the mixins when writing the tests because the full test suite takes ages then. My bad for committing this, and good spot!